shared/actions/chat/index.js

```diff
@@ -105,14 +105,19 @@ function* _incomingMessage(action: Constants.IncomingMessage): SagaGenerator<any
           return
         }
 
+        const conversationIDKey = Constants.conversationIDToKey(incomingMessage.convID)
         if (incomingMessage.conv) {
           yield call(Inbox.processConversation, incomingMessage.conv)
+        } else {
+          // Sometimes (just for deletes?) we get an incomingMessage without
```

Comment:
Is this really just a core bug?

Reply:
@mmaxim said there are a complicated set of reasons why deletes can't come with a conv object sometimes, so it's not an obvious bug, although it could be fixed in the future with some work.